### PR TITLE
docs: update examples and readme

### PR DESCRIPTION
**Pull request by conermurphy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,44 @@
-# Salable Web Components Svelte Demo
+# Salable Web Components : Svelte Demo
 
-This repository demonstrates how to set up Salable Web Components with Svelte.
-g
-## Getting started
+[Salable](https://www.salable.app) is a subscription licensing platform that allows users to have more control over their customers and how they interact with and use your applications/products.
```
Comment (adapta-sean) [inline README.md:3]: their

File: README.md [hunk 2]
```diff
@@ -1,17 +1,44 @@
...
+
+## Running the demo
+
+To run this demo locally with your own Salable account, follow the steps below.
+
+1. Clone this repository (`git clone https://github.com/Salable/Salable-Web-Components-Svelte-Demo.git`)
+2. Install the demo's dependencies (`npm install`)
+3. [Create a Salable account](https://salable.app/signup) (if you don't already have one)
+4. Create the required data for your chosen web component ([See requirements for each component here](https://docs.salable.app/web-components/web-components-latest/))
+5. Create a new session token for your chosen component using the Salable API
```
Comment (adapta-sean) [inline README.md:19]: The session token bit needs more information and/or a link to the api docs.

Comment (adapta-sean) [reply README.md:19]: At least link to the API docs and make a ticket to improve this for all readmes later.

File: README.md [hunk 3]
```diff
@@ -1,17 +1,44 @@
...
+6. Pass the required data to your component
```
Comment (adapta-sean) [inline README.md:20]: What data? Needs further explanation and/or links to relevant docs

File: README.md [hunk 4]
```diff
@@ -1,17 +1,44 @@
...
+
+- [Salable Documentation](https://docs.salable.app/)
+- [Salable Web Components Documentation](https://docs.salable.app/web-components/web-components-latest/)
+- [Svelte Documentation](https://svelte.dev/docs)
+
+## Want to give feedback? Or, found an issue?
+
+If you have found an issue or have feedback regarding this repo, please join our [Discord](https://discord.com/invite/xtytbyc4qz) and send us a message.
+
+If it's a quick fix like a misspelled word or broken link, feel free to create a pull request with the solution.
```
Comment (adapta-sean) [inline README.md:36]: I'll take any PRs if they fix shit. Make this more open.

**Review by orderandchaos-at-work — APPROVED** [review]

**Pull request merged**